libs/sdk-backend-tiger/src/convertors/toBackend/ObjRefConverter.ts

```diff
@@ -110,3 +111,16 @@ export function toMeasureValueFilterMeasureQualifier(ref: ObjRefInScope): LocalI
         );
     }
 }
+
+/**
+ * @internal
+ */
+export function toRankingFilterDimensionalityIdentifier(ref: ObjRefInScope): ObjectIdentifier {
+    if (isObjRef(ref)) {
+        return toObjQualifier(ref);
+    } else {
```

Comment:
i suggest to hide this limitation in the tiger backend impl by adding localId -> displayForm lookup.

otherwise the dev experience suffers: everywhere programmer can use the generated measure/attribute objects and the code hides that there may be references involved. this is an exception where "If you are on tiger and want to use ranking filter, send display form ref instead".

Reply:
The plan is to implement full support for localIds (for all MD object types) in Tiger GA. Then this limitation will be dropped from SDK code. I suggest to live with it until GA. Is it acceptable?

Reply:
yep makes sense. please create a followup so that we don't forget this

Reply:
Tiger follow-up: https://gitlab.com/gooddata/gdc-nas/-/issues/906